xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -3672,7 +3672,7 @@ extensions.actions.uninstallGlobally=Uninstall from farm
 extensions.actions.upgradeGlobally=Upgrade on farm
 extensions.actions.downgradeGlobally=Downgrade on farm
 extensions.actions.back=Back to list
-extensions.actions.continue=Continue
+extensions.actions.continue=Continue...
```

Comment:
Why do you think an ellipsis should be added here? This looks totally unrelated to the Jira issue.